themes/_default.yaml

```diff
@@ -505,6 +505,9 @@ sliding-panel:
     flex: "1 1 100%"
     overflow: "hidden"
     overflow-y: "auto"
+  footer:
+    border-top: "2px solid var(--tx-generic-color-secondary-darker)"
+    height: "80px"
```

Comment:
is it really different per brand?

Reply:
I guess people just using this file for defining component variables, not theme variables. A bit of misuse, but there is no better alternative, I'd say

Reply:
Basically, I agree with Atrem, I didn't find the better way to define values.